frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/state.ts

```diff
@@ -0,0 +1,89 @@
+export const diskModeDropdownItems = {
+  BLOCK: 'Block',
+  FILESYSTEM: 'Filesystem',
+};
+
+export const diskTypeDropdownItems = {
+  SSD: 'SSD / NVMe',
+  HDD: 'HDD',
+};
+
+export const diskSizeUnitOptions = {
+  TiB: 'TiB',
+  GiB: 'GiB',
+};
+
+export const initialState = {
+  volumeSetName: '',
+  storageClassName: '',
+  showNodesList: false,
+  diskType: diskTypeDropdownItems.SSD,
+  diskMode: diskModeDropdownItems.BLOCK,
+  maxDiskLimit: '',
+  nodeNames: [],
+  minDiskSize: 0,
+  maxDiskSize: 'All',
+  diskSizeUnit: 'TiB',
+  isValidMaxSize: true,
+  allNodeNames: [],
+};
+
+export type State = {
+  volumeSetName: string;
+  storageClassName: string;
+  showNodesList: boolean;
+  diskType: string;
+  diskMode: string;
+  maxDiskLimit: string;
+  nodeNames: string[];
+  minDiskSize: number | string;
+  maxDiskSize: number | string;
```

Comment:
Why are these tuple types?

Reply:
because max disk size can be 'All' too which is string.

Reply:
minDiskSize needs to have same type as its being compared. Feel free to check it out.